src/Stratis.Bitcoin/Connection/ConnectionManager.cs

```diff
@@ -90,6 +93,7 @@ public IReadOnlyNetworkPeerCollection ConnectedPeers
             IPeerAddressManager peerAddressManager,
             IEnumerable<IPeerConnector> peerConnectors,
             IPeerDiscovery peerDiscovery,
+            ISelfEndpointTracker selfEndpointTracker,
```

Comment:
I think this will always be null as it's not injected anywhere.

Reply:
This one is injected in BaseFeature.cs